client/examples/local_serving.py

```diff
@@ -1,17 +1,15 @@
-# import featureform as ff
-#
-# client = ff.ServingLocalClient()
-# fpf = client.features([("avg_transactions", "quickstart")],("CustomerID", "C1410926"))
-# print(fpf)
-#
-# training_set = client.training_set("fraud_training", "quickstart")
-# i=0
-# for r in training_set:
-#     print(r)
-#     i+=1
-#     if i > 10:
 import featureform as ff
 
+client = ff.ServingLocalClient()
+
+training_set = client.training_set("fraud_training", "quickstart")
+i=0
+for r in training_set:
```

Comment:
this might be a little cleaner if we do 
`for i, r in enumerate(training_set):
       print(r)
       if i > 10:
           break
`